src/uint.rs

```diff
@@ -108,11 +102,6 @@ impl UInt {
         UInt::from(ApInt::zero(width))
     }
 
-    /// Creates a new `UInt` with the given bit width that represents one.
-    pub fn one(width: BitWidth) -> UInt {
-        UInt::from(ApInt::one(width))
-    }
-
```

Comment:
The `one` constructor for `Uint` instances is without complications and should exist imo.